.releaserc

```diff
@@ -1,26 +1,19 @@
 {
+  "debug": true,
   "branches": [
     "main"
   ],
-  "verifyConditions": [
+  "plugins": [
+    "@semantic-release/commit-analyzer",
```

Comment:
These first two are automatically included in "plugins" if no other configured plugin implements their primary steps (analyzeCommits and generateNotes, respectively), but I added them explicitly just to be consistent with .releaserc files in the other languages.